packages/ui/src/components/pressable/pressable.native.tsx

```diff
@@ -62,24 +30,41 @@ interface PressableBaseProps extends RNPressableProps, RestyleProps {
    * <Pressable haptics={{ onPress: 'light', onLongPress: 'rigid' }} />
    */
   haptics?: PressableHapticFeedbackType | HapticConfig;
+  /**
```

Comment:
Do we need these comments? Maybe we could replace them with storybook examples

Reply:
These are JSDocs

Reply:
I get that but often comments become stale but in storybook it will break if props change. 

We had this chat before though and it's not a big deal for me, just _nit_-ing 😄